app/serializers/user_data_explorer_query_group_bookmark_serializer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class UserDataExplorerQueryGroupBookmarkSerializer < UserBookmarkBaseSerializer
+  attr_reader :data_explorer_query_group
+  attr_reader :data_explorer_query
```

Comment:
Are these required when there are methods defined with the same name?

Reply:
I think maybe Frank has just seen what I have done and copied that, but what I have done is kind of nonsensical :laughing: I don't think these are required, and I think I can probably remove similar from `UserChatMessageBookmarkSerializer` and `UserPostBookmarkSerializer `. Sorry!

Reply:
It's exactly as @martin-brennan says. I wasn't sure if they were actually necessary, but didn't check. I'll confirm they're not necessary and remove them. Thanks!